src/test/java/integration/EventsTest.java

```diff
@@ -79,6 +79,24 @@
 public class EventsTest {
 
     /**
+     * A branch name that contains a slash
+     */
+    private static final String SLASHY_BRANCH_NAME = "feature/ux-1";
+    /**
+     * The mangled name of {@link #SLASHY_BRANCH_NAME}
+     */
+    private static final String SLASHY_JOB_NAME = "feature-ux-1.0gorp7";
+    /**
+     * A branch name with unicode characters that have a two canonical forms (Korean for New Features).
```

Comment:
Why do we care about canonical forms? My understanding was that we only paid attention to the UTF-8 representation of a branch name as supplied by the SCM source.